app/jobs/search_keyword_job.rb

```diff
@@ -5,27 +5,24 @@ class SearchKeywordJob < ApplicationJob
 
   def perform(keyword_id)
     @keyword = Keyword.includes(:source).find(keyword_id)
-    search_result = search_service
+    search_result = call_search_service
     update_keyword(search_result)
   end
 
   private
 
-  def search_service
+  def call_search_service
     source_name = keyword.source.name.downcase
-    keyword_name = keyword.name
 
-    return Bing::SearchService.new(keyword_name).call if source_name == 'bing'
+    return Bing::SearchService.new(keyword.name).call if source_name == 'bing'
```

Comment:
Originally this would not have the lint error `Calls 'keyword.name' 2 times` 🤔 are you facing a different issue here?

Reply:
Following the logic, it is only called once. So we can skip it. Thanh is already commented about this.